### ISSUE TITLE
[2.8.26] Missing centreon-plugin-meta-2.8.26 in repo

### ISSUE THREAD

**Issue by a-foures** [centreon/centreon#6704]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.17

**OS**: RHEL 7.4

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Installed and updated with an offline repo

**Steps to reproduce the issue:**
1. Download repo from : http://yum.centreon.com/standard/3.4/el7/stable/
2. Configure for offline use
3. `yum centreon update`
    * update from 2.8.17 to 2.8.26

**Describe the results you received:**

```
--> Finished Dependency Resolution
Error: Package: centreon-plugin-meta-2.8.25-8.el7.centos.noarch (centreon-stable-noarch)
           Requires: centreon-common = 2.8.25-8.el7.centos
           Removing: centreon-common-2.8.17-10.el7.centos.noarch (@centreon-stable-noarch)
               centreon-common = 2.8.17-10.el7.centos
           Updated By: centreon-common-2.8.26-3.el7.centos.noarch (centreon-stable-noarch)
               centreon-common = 2.8.26-3.el7.centos

Error: Package: centreon-plugin-meta-2.8.25-8.el7.centos.noarch (centreon-stable-noarch)
           Requires: centreon-web = 2.8.25-8.el7.centos
           Removing: centreon-web-2.8.17-10.el7.centos.noarch (@centreon-stable-noarch)
               centreon-web = 2.8.17-10.el7.centos
           Updated By: centreon-web-2.8.26-3.el7.centos.noarch (centreon-stable-noarch)
               centreon-web = 2.8.26-3.el7.centos


```

**Describe the results you expected:** No dependency error

**Additional information you think important (e.g. issue happens only occasionally):**

There is no centreon-plugin-meta-2.8.26 rpm in http://yum.centreon.com/standard/3.4/el7/stable/noarch/RPMS/ and thus update is blocked ?

**Comment by a-foures** [comment]
I used `yum centreon update` as described in https://documentation.centreon.com/docs/centreon/en/2.8.x/upgrade/from_packages.html#update-components ...

With only `yum update` there is no dependency error.

**Comment by Hellnino18** [comment]
Hi,

I have almost the same problem, when I try on test : 

yum install centreon-2.8.24-1.el7.centos (I'm reinstalling my central server and I want to have the same version than the production).

--> Résolution des dépendances terminée
Erreur : Paquet : centreon-2.8.24-1.el7.centos.noarch (centreon-stable-noarch)
             Requiert : centreon-plugin-meta = 2.8.24-1.el7.centos

Package centreon-plugin-meta is obsolete and won't be installed because of  : 
Le paquet centreon-plugin-meta est rendu obsolète par centreon-plugin-Applications-Monitoring-Centreon-Central, mais l'obsolescence n'affecte pas les dépendances

And whit that : yum install centreon-2.8.24-1.el7.centos --skip-broken

It try to install 2.8.26...

Thanks for your help !

**Comment by lpinsivy** [comment]
@ganoze can you check RPM spec?

**Comment by ganoze** [comment]
@a-foures Indeed the update procedure is not complete. The `yum update` command should include all Centreon packages (to include _centreon-plugin-Applications-Monitoring-Centreon-Central_, see below). I just opened PR #6916 to fix this documentation issue.

@Hellnino18 Your issue is different and based on the way yum checks its dependencies. As far as I know you have to manually list all centreon package with the required version in your `yum install` command. If this feels like a bug to you please open another issue.

@all The reason behind these issues is that the package _centreon-plugin-meta_ was not used anymore. Therefore the _centreon-plugin-meta_ package is no longer provided and is obsoleted by _centreon-plugin-Applications-Monitoring-Centreon-Central_.
